tutor/src/flux/student-id.coffee

```diff
@@ -1,53 +0,0 @@
-{CrudConfig, makeSimpleStore, extendConfig, STATES} = require './helpers'
-{CourseStore} = require './course'
-_ = require 'lodash'
-
-ERROR_MAP = {
-  student_identifier_has_already_been_taken:
-    'The provided student ID has already been used in this course. ' +
-    'Please try again or contact your instructor.'
-  blank_student_identifier:
-    'The student ID field cannot be left blank. Please enter your student ID.'
-  no_change:
-    'You have not changed the student ID.  Please enter your new student ID and try again.'
-}
-
```

Comment:
Note that these api errors are no longer emitted - the BE now allows students to set ID to anything, so we don't need any special handling.